### PR TITLE
Add support for ZTT Maisijie device [49692]

### PR DESCRIPTION
**Pull request by ronz03130815**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
We really prefer to use the yaml way to implement new devices

File: includes/definitions/ztt.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+icon: ztt
+group: ZTT
+mib dir: ztt
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.49692
+            - .1.3.6.1.4.1.49692.1.
```
Comment (mpikzink) [inline includes/definitions/ztt.yaml:13]: You don't need line 13 to 17. Line 12 include this lines

**Comment by ronz03130815** [comment]
> We really prefer to use the yaml way to implement new devices

Yeah, understood, but seems not working in my side

File: includes/definitions/ztt.yaml (hunk 1 above)
Comment (ronz03130815) [reply includes/definitions/ztt.yaml:13]: Thanks

**Pull request closed without merging**